# バッテリ電圧SET
float64 value
---

